ex1/ex1/Exercises/Exercise1OOP.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ex1.Exercises
+{
+    public class Student
+    {
+        public string _name;
```

Comment:
There is a design rule that all fields should be declared as **private** and if you need to use these fields outside the class you should define corresponding public properties